src/main/resources/grouper.client.base.properties

```diff
@@ -525,7 +525,7 @@ esb.consumer.encryptionImplementation = edu.internet2.middleware.grouperClient.e
 ## general xmpp configuration
 grouperClient.xmpp.server.host = jabber.school.edu
 grouperClient.xmpp.server.port = 5222
-grouperClient.xmpp.user = username
+grouperClient.xmpp.user = uid
```

Comment:
Not sure if I commented this before, must've been on the UI PR, but revert this change back to username because this affects a different scope.